lib/wrappers/output.ts

```diff
@@ -0,0 +1,38 @@
+import fs, { PathLike } from "fs";
+import TransformNodeWrapper from "./transform-node";
+import { isAbsolute } from 'path';
+
+const logger = require('heimdalljs-logger')('broccoli:outputWrapper');
+
+const WHITELISTEDOPERATION = new Set([
+  'readFileSync',
+  'existsSync',
+  'lstatSync',
+  'readdirSync',
+  'statSync',
+  'writeFileSync',
+  'appendFileSync',
+  'rmdirSync',
+  'mkdirSync'
+]);
+
+export default function OutputWrapper (node: TransformNodeWrapper) {
+  return new Proxy(fs, {
+    get(target: any, propertyName: string): any {
+      return function() {
+        let [relativePath] = arguments;
```

Comment:
Instead of destructuring here, you can put `relativePath` in the function definition:

```js
return function(relativePath) {
```